test/test_torch.py

```diff
@@ -10840,7 +10840,7 @@ def test_copy_all_dtypes_and_devices(self, device):
         for dt in torch.testing.get_all_dtypes():
             x = torch.tensor([1, 2, 3, 4], dtype=dt, device=device)
             x_clone = x.clone()
-            if (device == 'cuda' and dt == torch.bfloat16):
+            if ('cuda' in device and dt == torch.bfloat16):
```

Comment:
I think `torch.device(device).type == 'cuda'` is more correct.

Reply:
Sure.